src/ui/components/Card/CardCore.tsx

```diff
@@ -69,6 +69,7 @@ export const CardCore = React.forwardRef<HTMLDivElement, CardViewProps>(
       isBuffedCrop = false,
       isSessionOwnersCard = false,
       showPlayCardButton = false,
+      playButtonDisabled = false,
```

Comment:
It doesn't seem this is being passed in as a prop anywhere. The `playButtonDisabled` prop needs to be passed as `true` when the game is in the setup state and the Field is full.